### DISCUSSION TITLE
Translation of nwg_shell_config-de_DE.json

### DISCUSSION THREAD

**Discussion by ricostap** [nwg-piotr/nwg-shell#442] [General]
Hi Piotr,

I have attached the file here. Did send you a couple of emails previously but not sure I got the correct email address.

HTH,
Rico
[nwg_shell_config-de_DE.json](https://github.com/user-attachments/files/17891575/nwg_shell_config-de_DE.json)

**Comment by nwg-piotr** [comment]
I haven't noticed any emails. Thank you very much. I've just [added](https://github.com/nwg-piotr/nwg-shell-config/commit/0cd89201fb98b5cba3753eab1d221bbf4b3af6b4) your file to the repo.

**Reply by ricostap** [reply]
Sorry It too so long, more than I thought. Do you have the other 2 files for translation in plain text maybe? I started to translate manually but then used AI and proof read manually. Better quality and faster.

**Reply by nwg-piotr** [reply]
We are not in a hurry.

You can download any file directly from GitHub repos. Just click a file name in the [langs](https://github.com/nwg-piotr/nwg-shell-config/tree/master/nwg_shell_config/langs) dir, and then select "Save link as" from over the "Raw" button.
